lib/payment_provider/handlers/async_handler.rb

```diff
@@ -11,6 +11,7 @@ class AsyncHandler
       # 'customer.subscription.created' => PaymentProvider::Handlers::SubscriptionHandler,
 
       def call(event)
+        ::Rails::logger.info("WEBHOOK: #{event.type} CONNECT: #{event.try(:account)} LIVEMODE: #{event.livemode}")
```

Comment:
info logs go to Rollbar ?

Reply:
No, they go to PaperTrail, only exceptions go to Rollbar.

Reply:
`Rollbar.info()` et al go to Rollbar obviously. But @thermistor you're saying Rails logger.info does not? @ivar I see your comment says "yes".

Reply:
Yes I am saying Rails::logger does not. Exceptions are sent to Rollbar automatically, and explicit calls to `Rollbar.notify('an exception or message')` of course also.

Reply:
Cool, guess I'm just wondering what @ivar 's "yes" is about?